crates/spirv-builder/Cargo.toml

```
@@ -16,12 +16,12 @@ skip-toolchain-check = ["rustc_codegen_spirv/skip-toolchain-check"]
 watch = ["notify"]
 
 [dependencies]
+rustc_codegen_spirv-types.workspace = true
+# See comment in lib.rs invoke_rustc for why this is here
+rustc_codegen_spirv = { workspace = true }
```

(see the comment on `Cargo.toml` for why this suggestion)
```suggestion
rustc_codegen_spirv = { workspace = true, default-features = false }
```